prow/config.yaml

```diff
@@ -6523,6 +6523,43 @@ postsubmits:
           secretName: service-account
 
 periodics:
+- name: ci-benchmark-scheduler-master
+  interval: 1h
+  agent: kubernetes
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20180102-0e2b24a0b-master
+      args:
+      - --repo=k8s.io/kubernetes=master
+      - --timeout=40
+      - --root=/go/src
+      env:
+      - name: GOOGLE_APPLICATION_CREDENTIALS
+        value: /etc/service-account/service-account.json
+      - name: USER
+        value: prow
+      - name: JENKINS_GCE_SSH_PRIVATE_KEY_FILE
+        value: /etc/ssh-key-secret/ssh-private
+      - name: JENKINS_GCE_SSH_PUBLIC_KEY_FILE
+        value: /etc/ssh-key-secret/ssh-public
```

Comment:
I think you don't need ssh keys for that integration test?

Reply:
Nice point.. Removed it.